components/meta-packages/history/history

```diff
@@ -999,6 +999,9 @@ network/dhcp/dhcpmgr/locale/zh_hk_tw@0.5.11-2013.0.0.0
 network/unison@2.27.57,5.11-0.170
 print/lp/filter/a2ps@4.14,5.11-2015.0.2.0 print/filter/a2ps@4.14,5.11-0.151.1.8.1
 print/mp@0.5.11,5.11-2018.0.0.0
+python/powerline@2.7-2020.0.1.1
+python/powerline-35@2.7-2020.0.1.1
+python/powerline-vim@2.7-2020.0.1.1
 redistributable@0.1,5.11-2015.0.2.0
```

Comment:
The versions you listed exist already. You'll need to increment the COMPONENT_REVISION (and add the build version: ,5.11):
python/powerline@2.7,5.11-2020.0.1.2
python/powerline-35@2.7,5.11-2020.0.1.2
python/powerline-vim@2.7,5.11-2020.0.1.2

Reply:
Did it, thanks.